R/primary-coding-suep.R

```diff
@@ -531,10 +551,10 @@ build_who_scale_suep_df <- function(trial_data, pid, docid, visitid){
     select(pid, .data$ecu_main_diag_covid)
   
   
-  # Date of last Treatment Update ---------------------------------------------------------------------
+  # Date of last Treatment Update ----------------------------------------------
   
   treatment_update_date <- trial_data$fuv3 %>%
-    left_join (select (trial_data$vp, .data$mnpvisid, visit_label_var_name), by = c("mnpvisid", visit_label_var_name)) %>%
+    left_join (trial_data$vp %>% select (.data$mnpvisid, visit_label_var_name), by = c("mnpvisid", visit_label_var_name)) %>%
     filter (.data$pr_check_treat.factor == "Ja") %>% #only treatment_update == YES
```

Comment:
do you need this date for every visit or do you need the last update date?

Reply:
this was your solution last summer ... we can talk about that